core/src/migrations/000077-addScheduleConfirmProfiles.ts

```diff
@@ -0,0 +1,15 @@
+export default {
+  up: async function (migration, DataTypes) {
+    await migration.sequelize.transaction(async () => {
+      await migration.addColumn("schedules", "confirmProfiles", {
+        type: DataTypes.BOOLEAN,
```

Comment:
This migration allows `schedules.confirmProfiles` to be null - is that intended?  I think it might be preferable to not allow it to be null, and in the migration, update all exiting rows to `false`

Reply:
Good catch!

Reply:
I think it's usually better to not assign a defaultValue in the migration.  This way you can still get an error from the database if a null is set, rather than silently assigning defaults - Check out https://github.com/grouparoo/grouparoo/blob/main/core/src/migrations/000060-addTypeToOption.ts

If you do a 2-part migration, first you add the column + a default for all existing rows, and then remove the defaults.